### PR TITLE
[V2 BetterMod] Patch Total Warns bug

### PR DESCRIPTION
**Pull request by FayeDel**
## Pull request type

- [ ] Feature addition
- [X] Bug fix
- [X] Grammar or minor corrections

## Does it overwrites current functionalities?

- [ ] Yes
- [X] No

## Did you test all of your changes?

- [X] Yes
- [ ] No

## Description of the changes

- [Fixed] A bug where [p]bcheck displays an embed case after all cases are deleted.
- [Minor} Fix grammatical errors and somewhat reworded descriptions for understanding purposes.

### CODE REVIEW

File: bettermod/bettermod.py [hunk 1]
```diff
@@ -832,7 +832,7 @@ def clear_cache(self):
 
     @thumbnail.command(pass_context=True, no_pm=True, name="ban", aliases="4")
     async def ban_thumbnail(self, ctx, url: str):
-        """Set the picture of the ban warning embed in modlog"""
+        """Set the picture of the ban warning embed in modlog".""
```
Comment (retke) [inline bettermod/bettermod.py]: There's a syntax error here.

Comment (DeltaXWizard) [reply bettermod/bettermod.py]: Noted, will fix.

File: bettermod/bettermod.py [hunk 2]
```diff
@@ -1757,16 +1758,17 @@ def clear_cache(self):
     @commands.group(pass_context=True, no_pm=True)
     @checks.mod_or_permissions(administrator=True)
     async def case(self, ctx):
-        """Edit warnings' reasons or remove them"""
+        """Edit or remove warning's reasons."""
```
Comment (retke) [inline bettermod/bettermod.py]: That's not clear; you edit a reason or remove a warning, but you don't remove a warning's reason.

File: bettermod/bettermod.py [hunk 3]
```diff
@@ -787,7 +787,7 @@ def clear_cache(self):
         if not url.endswith((".jpg", ".png", ".gif")) and not url.startswith(
             ("http://", "https://")
         ):
-            await self.bot.say("The URL given is not valid")
+            await self.bot.say("The URL given is not valid".)
```
Comment (retke) [inline bettermod/bettermod.py]: Another syntax error, the dot goes before the quotes.

**Comment by laggron42** [comment]
I no longer support V2. Feel free to advertise your fork around with your fixes if you still use this, but I don't want to continue with code from 3 years ago.

**Pull request closed without merging**